docs/plugins/google-cloud.md

```diff
@@ -221,32 +221,100 @@ Common dimensions include:
 - `topK` - the inference topK [value](https://ai.google.dev/docs/concepts#model-parameters).
 - `topP` - the inference topP [value](https://ai.google.dev/docs/concepts#model-parameters).
 
-### Flow-level metrics
+### Feature-level metrics
+
+Features are the top-level entry-point to your Genkit code. In most cases, this
+will be a Genkit flow, but if you do not use flows this will be the top-most
+span in a Genkit trace.
+
+Name                    | Description
+----------------------- | ---------------------------------------------------
+genkit/feature/requests | Counter for number of requests by feature.
+genkit/feature/latency  | Histogram tracking feature execution latency in ms.
+
```

Comment:
I wonder if we should do something like this in all the sections.

```suggestion
Name                    | Type      | Description
----------------------- | --------- | ---------------------------------------------------
genkit/feature/requests | Counter   | Number of requests.
genkit/feature/latency  | Histogram | Execution latency in ms.
```

Reply:
Good idea, updated!